google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -73,6 +73,15 @@ public static Mutation createUnsafe() {
     return new Mutation(true);
   }
 
+
+  /** Wraps the List of protobuf {@link com.google.bigtable.v2.Mutation}. */
```

Comment:
Please add a warning similar to createUnsafe:

It is the callers responsibility to make sure that all of the mutations have a deterministric timestamp. If a mutation has a serverside timestamp, then the mutation will no longer be idempotent, which might cause multiple duplicate values to be stored in Bigtable. This option should only be used for advanced usecases with extreme care.

Reply:
@igorbernstein2: done.